u-service-hello/src/main/java/com/gft/first/microservice_consul_1/MicroserviceConsul1Application.java

```diff
@@ -42,42 +42,66 @@ public String home() {
         return new ResponseEntity<>(message, HttpStatus.OK);
     }
 
-    @RequestMapping(value = "/api/inspect")
-    public String apiInspect(@RequestParam(value = "delay", required = false) Integer delay) {
+    @RequestMapping(value = "/api/inspect", method= RequestMethod.GET )
+    public String apiInspectCsv(HttpServletResponse response,
+                                                  @RequestParam(value = "delay", required = false) Integer delay,
+                                                  @RequestParam(value = "format", required = false) String format) throws IOException {
+
+        if (format != null && format.equals("Fcsv")) {
```

Comment:
FYI ideally this should be refactored to the strategy pattern, but leave it as is